packages/gafl-webapp-service/src/pages/contact/address/entry/address-entry.njk

```diff
@@ -19,19 +19,19 @@
 {%
     set errorMap = {
         'premises': {
-            'string.empty': { ref: '#premises', text: 'You did not enter your building number or name' },
-            'string.max': { ref: '#premises', text: 'Your building number or name must be less than 50 characters' }
+            'string.empty': { ref: '#premises', text: 'Enter the building number or name' },
```

Comment:
Inconsistent - we use "house number or name" below and "building number or name" here

Reply:
See above